Documentation/ceph-upgrade.md

```diff
@@ -416,6 +416,59 @@ updated we wait for things to settle (monitors to be in a quorum, PGs to be clea
 MDSes, etc.), then only when the condition is met we move to the next daemon. We repeat this process
 until all the daemons have been updated.
 
+### Disable `bluestore_fsck_quick_fix_on_mount`
+> **WARNING: There is a notice from Ceph for users upgrading to Ceph Pacific v16.2.6 or lower from
+> an earlier major version of Ceph. If you are upgrading to Ceph Pacific (v16), please upgrade to
```

Comment:
It's specifically from octopus, right? I thought the setting didn't apply to nautilus
```suggestion
> Ceph Octopus v15. If you are upgrading to Ceph Pacific (v16), please upgrade to
```

Reply:
It does apply to Nautilus. The setting was introduced in v14.2.5.